### ISSUE TITLE
[BUG] TMDB Helper or AH2 lose focus running A4kSubtitles addon

### ISSUE THREAD

**Issue by TechXXX** [jurialmunkey/skin.arctic.horizon.2#612]
### Skin section

Other

### Current Behavior

As mentioned in the title when I activate subtitle search via the menu (important) and the results pop up it loses focus right away or after a short time (seconds). If on a remote it feels like a freeze until you click back but with a mouse you can regain focus and it works as normal. (Dutch language in pics - Ondertiteling = Subs). If you run a4ksubs a second time the problem is gone. Another thing, I have a keymapping for search subtitles (download), when I use that there also isn't a problem. 

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/10571873/200301771-764fbde4-5299-463e-bb71-b4af3565a5e4.png">

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/10571873/200302283-629d993e-30f3-4bf9-b28c-4627d01f7e44.png">

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/10571873/200302595-60f2d76a-2899-4d45-ba18-d1e448a46605.png">

Just tried it with kodi's default skin and the problem went away, must be something with tmdb helper and ah2. 

### Expected Behavior

When you activate  sub search you get a list of subs, you're supposed to scroll them at your leisure, now it freezes after seconds, without a mouse you have to click back to regain control. 

### Steps To Reproduce

in AH2 search for movies, or open TMDB helper from Addons and start any movie - click enter for OSD - goto sub icon and enter - download subtitle - wait 2/3 seconds - try to navigate the list of subs....should freeze in a few seconds - unfreeze with mouse or back. 

### Screenshots and Additional Info

I just tried it with a locally scraped movie and it works fine there, so it's movies scraped via the players url in tmdb helper. 

Update: Not having the problem on my macos version of kodi 19.4, on windows and shield the problem persists...not sure why it went away on my mac. Maybe it's a third addon problem. Adding the debug log of the windows machine exited right after bug. 

[kodi.log](https://github.com/jurialmunkey/skin.arctic.horizon.2/files/9951449/kodi.log)

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by tfouto** [comment]
This also happens to me. The same exact behaviour.

**Comment by TechXXX** [comment]
Thanks a bunch Jurial! Keep up the good work, bought you a cup of coffee.

**Comment by jurialmunkey** [comment]
> Thanks a bunch Jurial! Keep up the good work, bought you a cup of coffee.

@TechXXX - Thanks! 😁 
Let me know if the problem still happens. I'm not 100% sure what the underlying cause is but fairly sure I've managed to work around it. You might need to double press sometimes to get the focus back, but shouldn't get completely stuck like before where only way out was pressing back. I haven't been able to recreate that problem since I added the workaround.
